custom/ewsghana/reports/specific_reports/reporting_rates.py

```diff
@@ -154,7 +154,7 @@ def headers(self):
     @property
     def rows(self):
         rows = []
-        if self.location_id:
+        if self.location_id and self.get_locations:
```

Comment:
I know you didn't write it, but it's pretty confusing to have a property start with `get_`. boy scout rule and what not